champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/ArcticArmour.java

```diff
@@ -199,9 +201,21 @@ public float getEnergy(int level) {
         return (float) (energy - ((level - 1) * energyDecreasePerLevel));
     }
 
+    @Override
+    public float getEnergyStartCost(int level) {
+        return (float) (energyStartCost - ((level - 1) * energyStartCostDecreasePerLevel));
+    }
```

Comment:
Seeing as this is the same across all, I propose just overriding it in ActiveToggleSkill

Reply:
I completely missed that ActiveToggleSkill extends skill (and therefore has access to the variables)